apypie/api.py

```diff
@@ -66,15 +71,18 @@ def __init__(self, **kwargs):
 
     @property
     def apidoc(self):
+        # type: () -> dict
         if self._apidoc is None:
             self._apidoc = self._load_apidoc()
         return self._apidoc
 
     @property
     def apidoc_cache_file(self):
+        # type: () -> str
         return os.path.join(self.apidoc_cache_dir, '{0}{1}'.format(self.apidoc_cache_name, self.cache_extension))
 
     def _cache_dir_contents(self):
+        # type: () -> Iterable
```

Comment:
I think this returns [PathLike](https://docs.python.org/3/library/os.html#os.PathLike) in the iterable, at least on 3.6+

Reply:
No, `glob` returns strings.